fili-core/src/main/java/com/yahoo/bard/webservice/web/DataApiRequest.java

```diff
@@ -93,6 +95,7 @@
     private static final Logger LOG = LoggerFactory.getLogger(DataApiRequest.class);
     private static final String RATIO_METRIC_CATEGORY = "Ratios";
     public static final String REQUEST_MAPPER_NAMESPACE = "dataApiRequestMapper";
+    private static final String DATE_TIME_STRING = "dateTime";
```

Comment:
It's obvious from the type signature and the value that this is a date time string. Could we get a better name for this variable that indicates the _meaning_ of it's contents?

/CanBeInASubsequentPr